woodwork/deserializers/utils.py

```diff
@@ -26,16 +26,35 @@
 
 
 def get_deserializer(
-    path, filename, data_subdirectory, typing_info_filename, profile_name
+    path,
+    filename,
+    data_subdirectory,
+    typing_info_filename,
+    profile_name,
+    format,
 ):
     """Determine the proper Deserializer class to use based on the specified parameters.
     Initializes and returns the proper Deserializer object."""
```

Comment:
Would also be nice to add `Args` and `Returns` info do docstring here

Reply:
I can add those. Also, that made me think that this should probably be a private function. I would not ever expect a user to call this function and interact with the deserializer object that is returned.

Reply:
Updated [efc0cd8](https://github.com/alteryx/woodwork/pull/1377/commits/efc0cd89032b4253fb601fde6b5f4c5c315dfbc7)